src/components/Breadcrumbs/Breadcrumbs.vue

```diff
@@ -139,6 +140,9 @@ export default {
 		window.addEventListener('resize', debounce(() => {
 			this.handleWindowResize()
 		}, 100))
+		subscribe('navigation-toggled', () => this.$nextTick(() => {
+			setTimeout(() => this.handleWindowResize(), 110)
```

Comment:
The timeout here is necessary since the app-navigation toggle is animated with 100 ms duration.